### DISCUSSION TITLE
External go2rtc stream flashing green

### DISCUSSION THREAD

**Discussion by calabresemic** [blakeblackshear/frigate#14244] [Ask A Question]
I'm using the Abode Camera Streaming add-on for Home Assistant and running Frigate on an external server. I have the rtsp stream from go2rtc in that add-on loaded into Blue Iris and Frigate and after frigate has been running for maybe 5-10 hours it starts flashing green and dropping frames. 

Blue Iris and VLC both show the stream fine without dropping frames so I'm not sure what is happening. I have enabled debug logging in Frigate and I don't see anything for that camera other than frigate.record.maintainer and peewee. I have tried putting the rtsp feed through go2rtc on Frigate and just using it straight as a camera input. I have also tried to use ?mp4 and just using the straight feed.

If anyone has any ideas on what I'm doing wrong or what's going on with this it would be greatly appreciated.

![Screenshot 2024-10-09 084102](https://github.com/user-attachments/assets/ec1cbc88-bf9c-496d-a957-abe5a772e1b2)

Here's my config in frigate for that camera:
``` yaml
garage:
    ffmpeg:
      inputs:
        - path: rtsp://192.168.X.X:8554/spy_cam?mp4
          input_args: preset-rtsp-restream
          roles:
            - record
            - detect
```

Here's the info from go2rtc on the add-on:
``` json
{
  "producers": [
    {
      "type": "WebRTC/Kinesis active producer",
      "remote_addr": "udp4 host 10.10.200.102:52019",
      "medias": [
        "video, recvonly, H264",
        "audio, recvonly, OPUS/48000/2"
      ],
      "receivers": [
        "101 OPUS/48000/2, bytes=2726623680, senders=1",
        "97 H264, bytes=36304043015, senders=2"
      ],
      "recv": 40058053779
    }
  ],
  "consumers": [
    {
      "type": "RTSP passive consumer",
      "url": "rtsp://192.168.X.X:8554/spy_cam?video=all\u0026audio=all",
      "remote_addr": "192.168.X.X:64061",
      "user_agent": "BlueIris",
      "sdp": "v=0\r\no=- 1 1 IN IP4 0.0.0.0\r\ns=go2rtc/1.8.4\r\nc=IN IP4 0.0.0.0\r\nt=0 0\r\nm=audio 0 RTP/AVP 96\r\na=rtpmap:96 OPUS/48000/2\r\na=fmtp:96 minptime=10;useinbandfec=1\r\na=control:trackID=0\r\nm=video 0 RTP/AVP 97\r\na=rtpmap:97 H264/90000\r\na=fmtp:97 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42e01f\r\na=control:trackID=1\r\n",
      "medias": [
        "audio, sendonly, ALL",
        "video, sendonly, ALL"
      ],
      "senders": [
        "96 OPUS/48000/2, bytes=2726623680, receivers=1",
        "97 H264, bytes=36304043015, receivers=1"
      ],
      "send": 39853513066
    },
    {
      "type": "RTSP passive consumer",
      "url": "rtsp://192.168.X.X:8554/spy_cam?mp4",
      "remote_addr": "192.168.X.X:58428",
      "user_agent": "FFmpeg Frigate/0.14.1-f4f3cfa",
      "sdp": "v=0\r\no=- 1 1 IN IP4 0.0.0.0\r\ns=go2rtc/1.8.4\r\nc=IN IP4 0.0.0.0\r\nt=0 0\r\nm=video 0 RTP/AVP 96\r\na=rtpmap:96 H264/90000\r\na=fmtp:96 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42e01f\r\na=control:trackID=0\r\n",
      "medias": [
        "video, sendonly, H264, H265",
        "audio, sendonly, MPEG4-GENERIC"
      ],
      "senders": [
        "96 H264, bytes=5850731784, receivers=1"
      ],
      "send": 5937748184
    }
  ]
}
```

**Comment by NickM-27** [comment]
If it shows this on the main live view and the snapshot / debug view then that is what the camera is sending

**Comment by eccentricexit** [comment]
Started getting this as well. Occasionally the bottom half of the stream turns green

Top of stream cropped.
![image](https://github.com/user-attachments/assets/2db84d93-e4b7-4732-9705-393b3a9148be)


Interestingly it looks like some of the data is going through... it is not full green, there are some details
